template/src/index.js

```diff
@@ -3,16 +3,24 @@ import 'regenerator-runtime/runtime'
 import {registerHtml, start} from 'tram-one'
 import ColorHeader from './components/ColorHeader'
 import './styles.css'
+import { useColor } from './hooks/ColorHook'
 
 const html = registerHtml({
 	ColorHeader
 })
 
 const home = () => {
+  const [color, incrementColor] = useColor()
```

Comment:
is the indentation correct here? I feel like it should be lined up with the `return html`, I also know that the indentation is a bit wonky because of the tabs here vs what's used for the tagged template literals